README.md

```diff
@@ -34,6 +34,10 @@ This library provides a [socket factory][socket-factory] for use with the
 1. Configure a connection pool that configures the driver to use the Connector
    as a socket factory
 
+For usage information specific to your database engine and driver, see the pages below:
```

Comment:
AlloyDB will always be Postgres only. And since we only support JDBC right now, perhaps we say, "For information on configuring a connection, see" or something generic like that.

Reply:
Updated. Thanks!